src/gamekit.cs

```diff
@@ -1762,17 +1769,17 @@ interface GKDialogController {
 		[Export ("sharedDialogController")]
 		GKDialogController SharedDialogController { get; }
 	}
-#endif
 
 	[NoWatch]
 #if MONOMAC
 	[Deprecated (PlatformName.MacOSX, 10, 12)]
+	[NoiOS][NoMacCatalyst][NoTV]
```

Comment:
The availability attributes should be the same on all platforms for the type, so they should be brought outside of the  `#if MONOMAC` condition.